### PR TITLE
Update NuGet Packages metadata

### PR DESCRIPTION
**Pull request by PatoBeltran**
## Bug
Fixes: https://github.com/NuGet/Home/issues/6518

Update NuGet packages metadata to have the correct copyright

### CODE REVIEW

**Review by rohit21agrawal — APPROVED** [review]

**Review by jainaashish — CHANGES_REQUESTED** [review]
One question, why should ProjectUrl be pointing to `NuGet.Home`? IMO it should be `nuget.org`

**Comment by nkolev92** [comment]
@jainaashish 
Why NuGet.org?

These are client libraries, not the server. 

Redirecting to NuGet/NuGet.Client instead of NuGet/Home might make a bit more sense.

**Review by nkolev92 — APPROVED** [review]
LGTM. 

I disagree with @jainaashish that the projectUrl should be nuget.org.

**Comment by jainaashish** [comment]
@nkolev92 this is the project url we're talking about and not repository url... and there is only single NuGet, not client nuget and server nuget for outside world.

**Comment by nkolev92** [comment]
NuGet.org still doesn't make sense. 

People are aware there's a client and a server. 

ProjectUrl should ideally be a place where people can file issues/ask for help or look at documentation, which I guess in that context it makes sense for it to stay nuget/home.

**Comment by jainaashish** [comment]
I disagree ProjectUrl is the place to get more details about the project and not your github link to file issues. Nor i've seen this usage of ProjectUrl any where else. Look at some of the good packages even on nuget.org and what they add as projecturl...

Although we can add our github link on nuget.org to allow customers to move to source code page from there.

**Comment by mishra14** [comment]
@PatoBeltran What is the reasoning behind changing the copyright? If I remember correctly, NuGet client is still under .NET foundation. cc @karann-msft

**Comment by PatoBeltran** [comment]
@mishra14 you can look at the issue https://github.com/NuGet/Home/issues/6518

@jainaashish @nkolev92 Given that this is for packages, the project url should be somewhere where you can find more details about the project (eg. documentation, reference to where to file issues, specs, etc.). Since we don't have one page that has everything I think either NuGet/Home or nuget.org are acceptable, but NuGet/Home seems to be more closely couple to us.

**Comment by nkolev92** [comment]
@PatoBeltran you can probably merge this.

**Comment by mishra14** [comment]
@nkolev92 @PatoBeltran  before merging we should clarify the copyright issue with @karann-msft or @anangaur

**Comment by nkolev92** [comment]
@mishra14 doesn't https://github.com/NuGet/Home/issues/6518#issuecomment-403654822 answer that?

**Comment by nkolev92** [comment]
Furthermore there's some more changes required per https://github.com/NuGet/Home/issues/6518#issuecomment-408205989
@PatoBeltran

**Comment by mishra14** [comment]
@nkolev92 We discussed this offline right? Based on what I remember the Gallery (and the server) team moved to Microsoft, but the client team has not made that transition. We should clarify that before changing our copyright.

**Comment by nkolev92** [comment]
//cc @karann-msft

**Comment by PatoBeltran** [comment]
updated with further required changes. @karann-msft can you please review?

File: build/config.props [hunk 1]
```diff
@@ -42,8 +42,8 @@
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/NuGet/NuGet.Client</RepositoryUrl>
     <PackageTags>nuget</PackageTags>
-    <Description>NuGet client library.</Description>
-    <Copyright>&#169; .NET Foundation. All rights reserved.</Copyright>
+    <Description>NuGet client library. Microsoft holds the copyright for this NuGet package. .NET Foundations holds the copyright to the source.</Description>
```
Comment (karann-msft) [inline build/config.props:45]: As long as `&#169;` translates to © when the package is published, it looks good.

File: src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec [hunk 2]
```diff
@@ -8,8 +8,8 @@
     <licenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</licenseUrl>
     <projectUrl>https://aka.ms/nugetprj</projectUrl>
     <iconUrl>https://raw.githubusercontent.com/NuGet/Media/master/Images/MainLogo/256x256/nuget_256.png</iconUrl>
-    <copyright>&#169; .NET Foundation. All rights reserved.</copyright>
-    <description>NuGet Command Line Tool</description>
+    <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
```
Comment (karann-msft) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec:11]: As long as `&#169;` translates to © when the package is published, it looks good.

**Review by karann-msft — APPROVED** [review]
LGTM.

**Review by jainaashish — APPROVED** [review]

**Comment by karann-msft** [comment]
:shipit:

**Pull request merged**